server/database/models/donation.ts

```diff
@@ -14,6 +19,18 @@ const Donation = sequelize.define('donations', {
     type: DataTypes.INTEGER,
     allowNull: true,
   },
+  description: {
+    type: DataTypes.STRING,
+    allowNull: true,
+  },
+  deliver_time: {
+    type: DataTypes.DATE,
+    allowNull: false,
+  },
+  location: {
+    type: DataTypes.STRING,
+    allowNull: true,
```

Comment:
AllowNull by default is true so no need to add them, and is it ok for the location to be null?